src/components/flex-container/flex-container.tsx

```diff
@@ -10,9 +10,13 @@ import { Component, h, Prop } from '@stencil/core';
  * Lime Elements. Please use CSS for your flexible container needs 🙂
  * https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox
  *
+ * :::warning
+ * We will delete this component soon!
+ * Do not use it.
+ * :::
```

Comment:
It's already marked as deprecated, and the description has already been changed to say that it will be removed. I don't think we need this extra warning, and I think it sets a precedent we don't want. Marking something as deprecated should be enough, we shouldn't teach people to expect that there will also be a big red warning label before we remove a component. (Also, no one will actually see this warning, since the component has been marked as private, so it's a bit of a moot point. I still wanted to explain my reasoning though 🤗)

<img width="945" alt="image" src="https://user-images.githubusercontent.com/154725/210376576-eee4be27-a878-47bd-9c6b-9104e4cc5175.png">

Reply:
⚡